app/jobs/order_cycle_closing_job.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-class OrderCycleClosingJob < ActiveJob::Base
+class OrderCycleClosingJob < ApplicationJob
```

Comment:
We should probably drop this for now, as we don't currently have an `ApplicationJob` class. You can ignore the linter in this case :+1:

Reply:
Thank you for your advice @Matt-Yorkley 
I just removed it. And could you give me a few minutes to squash my commits, I assume I will have to do so...🙏